### ISSUE TITLE
cant update to mg 3.81

### ISSUE THREAD

**Issue by damian-666** [MonoGame/MonoGame.Samples#59]
before i just had to install net 3.1 ro something.. ive uninstalled all taht.. 

it wont resolve the xna namespace..  in the netstandard. i can buld one platfrom but i cant share code form desktop to droid

**Comment by harry-cpp** [comment]
As stated in the referenced issue, you need to install .net 6 (and VS 2022). Create fresh project to see how the project file and dotnet tools need to be setup.

**Comment by damian-666** [comment]
ok I have used upgate-assistant and it did alot of this for me.  change
netstandard 2.0 and even update MG version..  except  I needed to add the
.config folder that was needed, i i had to add the content thing.  for
desktop DX and GL worked fine.  and i thnik faster and more stable   ...
Except on Android where it failed to fix it , so i had to merge in from the
new template.   and stillhaving one last weird issue so cant qute build it
yet  ..  But even the templaatle and simple code is still using xamarin
just for the launcher, so im going to try the Maui  pareview feature later
and wait this out...  it mght be too early to try ths out but would be huge
not waiting 3 min to deploy and build a simple tiny app.

 my shader sample runs fine  but the build deploy is dog slow....and the
forms code i think not much point to deal what it now.  they are going fast
this way and,  im not going to use  their  new UI, just wanna ditch the
xamarin tool chain.

 now to get myh WPF - MG app working, i think that will work...craftwork
has his on netcore.. 3.1 so it should pan out thanks..
